src/dmd/builtin.d

```diff
@@ -373,14 +373,17 @@ public extern (C++) void builtin_init()
     add_builtin("_D4core4math6rndtolFNaNbNiNfeZl", &eval_unimp);
     // @safe @nogc pure nothrow real function(real)
     add_builtin("_D3std4math3tanFNaNbNiNfeZe", &eval_tan);
+    add_builtin("_D3std4math4trig3tanFNaNbNiNfeZe", &eval_tan);
     add_builtin("_D3std4math5expm1FNaNbNiNfeZe", &eval_unimp);
     // @trusted @nogc pure nothrow real function(real)
     add_builtin("_D3std4math3tanFNaNbNiNeeZe", &eval_tan);
+    add_builtin("_D3std4math4trig3tanFNaNbNiNeeZe", &eval_tan);
     add_builtin("_D3std4math3expFNaNbNiNeeZe", &eval_exp);
     add_builtin("_D3std4math5expm1FNaNbNiNeeZe", &eval_expm1);
     add_builtin("_D3std4math4exp2FNaNbNiNeeZe", &eval_exp2);
     // @safe @nogc pure nothrow real function(real, real)
     add_builtin("_D3std4math5atan2FNaNbNiNfeeZe", &eval_unimp);
+    add_builtin("_D3std4math4trig5atan2FNaNbNiNfeeZe", &eval_unimp);
```

Comment:
Please bump the `builtins._init(65);` on line 334 as well. 

I'd love to get rid of this whole section as well at some point.

Reply:
A quick count shows that there are already 76 entries without patch.

Reply:
Bumped to 113 (expected number of entries with both `std.math` and `std.math.xxx`).